apps/web/src/contexts/ABTestingContext/ABTestingContext.tsx

```diff
@@ -0,0 +1,40 @@
+import { FEATURE_FLAGS } from 'contexts/ABTestingContext/config'
+import { createContext, useCallback, useContext, useState } from 'react'
+
+interface ABTestingProviderProps {
+  children: React.ReactNode
+  ABTestUserResults: { [flag in FEATURE_FLAGS]: boolean }
+}
+
+type ABTestingContextType = {
+  featureFlagResults: { [flag in FEATURE_FLAGS]: boolean }
+  selectUserfeatureABResult: (flagKey: FEATURE_FLAGS) => boolean
+}
```

Comment:
The context only need to provide the data. As for the getter function can be defined inside a lower level custom hook, e.g. inside a hook called `useExperimentalFeatureEnabled`

```suggestion
type ExperimentalFeatureContext = {
  [feature in EXPERIMENTAL_FEATURES]: boolean
}
```